toolkit/tools/pkg/isomakerlib/isomaker.go

```diff
@@ -76,18 +81,22 @@ func NewIsoMaker(unattendedInstall bool, baseDirPath, buildDirPath, releaseVersi
 		imageNameBase:      imageNameBase,
 		imageNameTag:       imageNameTag,
 	}
+
+	return isoMaker, nil
 }
 
-func NewIsoMakerWithConfig(unattendedInstall, enableBiosBoot bool, baseDirPath, buildDirPath, releaseVersion, resourcesDirPath string, config configuration.Config, initrdPath, grubCfgPath, isoRepoDirPath, outputDir, imageNameBase, imageNameTag string) *IsoMaker {
+func NewIsoMakerWithConfig(unattendedInstall, enableBiosBoot bool, baseDirPath, buildDirPath, releaseVersion, resourcesDirPath string, config configuration.Config, initrdPath, grubCfgPath, isoRepoDirPath, outputDir, imageNameBase, imageNameTag string) (isoMaker *IsoMaker, err error) {
 
 	if imageNameBase == "" {
 		imageNameBase = defaultImageNameBase
 	}
 
-	// verifyConfig() panics if an error occurs.
-	verifyConfig(config, unattendedInstall)
+	err = verifyConfig(config, unattendedInstall)
+	if err != nil {
+		return nil, err
```

Comment:
Purely informal, no need to change anything, if you prefer this format: with the named output, you can now simplify all returns like so.
```suggestion
		return
```

Reply:
I'll leave the explicit returns for now, Pawel. I find the explicitness to be more readible.